src/Confluent.SchemaRegistry.Serdes.Avro/AvroSerializerConfig.cs

```diff
@@ -49,8 +52,27 @@ public static class PropertyNames
             ///     default: true
             /// </summary>
             public const string AutoRegisterSchemas = "avro.serializer.auto.register.schemas";
+
+            /// <summary>
+            ///     The subject name strategy to use for schema registration / lookup.
```

Comment:
yep. this is a minor point however since no one should use these config property names (they should bind directly to the class property names and use these in their config) - it's just done like this for consistency with producer, consumer etc.